### PR TITLE
test: add output reporting to cypress

### PR DESCRIPTION
**Pull request by marcelltoth**
Adds reporting of test results, screenshots & videos to Cypress.

Needed for stoplightio/elements#670

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,6 +8,7 @@ node_modules
 # testing
 /coverage
 packages/*/coverage
+cypress/results
```
Comment (mallachari) [inline .gitignore:11]: There's also `cypress/videos` directory created. Might be good to add it too.

**Review by mallachari — APPROVED** [review]
LGTM. Nice video!

File: .gitignore (hunk 1 above)
Comment (marcelltoth) [reply .gitignore:11]: Is there one? Isn't it from one of your old runs?

I intentionally redirected the videos to inside the `cypress/results/videos` folder so we keep everything at one place.

I guess you ran e2e before which created that folder but if it's not the case, LMK.

**Comment by marcelltoth** [comment]
@mallachari I merge this because this is important for me to be able to progress with stoplightio/elements#670, but if you still have the `cypress/videos` folder spawning automatically LMK and I'll jump on it.

File: .gitignore (hunk 1 above)
Comment (mallachari) [reply .gitignore:11]: Yeah, that's probably that, I'll check in the moment but feel free to merge it already

Comment (mallachari) [reply .gitignore:11]: That was it indeed :man_facepalming:

**Pull request merged**